cirq/_doc.py

```diff
@@ -44,4 +46,20 @@ def document(value: Any, doc_string: Optional[str] = None):
     """
     docs = DocProperties(doc_string=doc_string)
     RECORDED_CONST_DOCS[id(value)] = docs
+
+    ## this is how the devsite API generator picks up
+    ## docstrings for type aliases
+    if sys.version_info >= (3, 7):
+        try:
+            value.__doc__ = doc_string
+        except AttributeError:
+            # we have a couple (~ 7) global constants of type list, tuple, dict,
+            # that fail here as their __doc__ attribute is read-only.
+            # For the time being these are not going to be part of the generated
+            # API docs. See b/166663013 for more info.
```

Comment:
a bit strange to have b/XX in external repo.  maybe make a cirq issue and the include this reference there?

Reply:
Fixing!